gbn/gbn_conn.go

```diff
@@ -118,6 +121,14 @@ func newGoBackNConn(ctx context.Context, sendFunc sendBytesFunc,
 
 	ctxc, cancel := context.WithCancel(ctx)
 
+	// Construct a new prefixed logger.
+	identifier := "client"
+	if isServer {
+		identifier = "server"
+	}
+	prefix := fmt.Sprintf("(%s)", identifier)
+	plog := build.NewPrefixLog(prefix, log)
```

Comment:
Thanks a lot for this :tada::pray:!!! This will be a really, really helpful when investigating the logs to discover future potential issues 🔥! I previously had to implement some local prefix solution, to know which logs were from the server, and which ones were from the client, when investigating the packeting bug 😅. So this would have helped a lot!

Reply:
awesome 🎉